assets/app/scripts/controllers/projects.js

```diff
@@ -24,6 +24,45 @@ angular.module('openshiftConsole')
       $('#openshift-logo').off('click.projectsPage');
     });
 
+    $scope.toggleDelete = function(project) {
+      // to toggle prompt to delete
+      // disable the delete button if current project is being deleted
+      if (!$scope.projects[project]['deleting']) {
```

Comment:
store the toggled/deleting state in separate maps... the `projects` hash contains API objects we don't want to mutate